### PR TITLE
Remove post-submit jobs

### PR DESCRIPTION
**Pull request by bnrjee**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: jobs/aws/eks-distro/main-1-18-postsubmits.yaml
Comment (TerryHowe) [inline jobs/aws/eks-distro/main-1-18-postsubmits.yaml:42]: where are we getting the artifact bucket now?

Comment (TerryHowe) [reply jobs/aws/eks-distro/main-1-18-postsubmits.yaml:42]: Maybe you don't need the bucket, you just need the web site

Comment (bnrjee) [reply jobs/aws/eks-distro/main-1-18-postsubmits.yaml:42]: Artifacts bucket will be added as an env in the eks-distro main makefile.

Comment (TerryHowe) [reply jobs/aws/eks-distro/main-1-18-postsubmits.yaml:42]: Okay, I have something going there. We'll see how it is received.

**Comment by bnrjee** [comment]
Putting it on hold. It should go just before merge,
https://github.com/aws/eks-distro/pull/139

/hold

File: jobs/aws/eks-distro/main-1-18-postsubmits.yaml
Comment (vivek-koppuru) [reply jobs/aws/eks-distro/main-1-18-postsubmits.yaml:42]: Why don't we actually have it as an env here so we don't have it as a default for customers in the Makefile?

File: jobs/aws/eks-distro/main-1-18-postsubmits.yaml
Comment (vivek-koppuru) [inline jobs/aws/eks-distro/main-1-18-postsubmits.yaml:41]: If we are storing the role name here, don't we want the account id also? Or should we instead just store the role arn instead?

**Review by vivek-koppuru — CHANGES_REQUESTED** [review]

File: jobs/aws/eks-distro/main-1-18-postsubmits.yaml
Comment (vivek-koppuru) [reply jobs/aws/eks-distro/main-1-18-postsubmits.yaml:42]: Can we also have the main presubmit invoke something like a presubmit target rather than what it is right now? That way, we can have it invoked on the `cmd/` directory as well.

**Review by vivek-koppuru — APPROVED** [review]
/lgtm

**Pull request closed without merging**
